LEAF_Request_Portal/sources/FormWorkflow.php

```diff
@@ -1497,10 +1497,10 @@ private function getFields(): array
                     }
                     break;
                 case (str_starts_with($format, "checkboxes") != false):
+                    $data = $this->buildCheckboxes(unserialize($data));
+                    break;
                 case (str_starts_with($format, "multiselect") != false):
-                    if ($this->isJsonString($data) && is_array(json_decode($data))) {
-                        $data = $this->buildMultiselect(json_decode($data));
-                    }
+                    $data = $this->buildMultiselect(unserialize($data));

```

Comment:
Removing checks makes me nervous.  What happens if it gets something that isn't json?  How badly will it fail?

Reply:
This check was causing the fail in the first place. I don't remember why this is checking for JSON. This is only ever serialized PHP arrays, which is a string in this context.